### ISSUE TITLE
nethserver-freepbx: minor fixes and enhancements for RC

### ISSUE THREAD

**Issue by Stell0** [NethServer/dev#5268]
- [X]  Opern RTP ports if SIPS or WebRTC are enabled from interface (add it only once on shorewall config)

- [X] Access list to FreePBX interface from NethServer web UI

- [X] Don't open RTP 10000:20000 with nethserver-janus: they already belong to asterisk

- [X] Add asterisk manager port 5038 to Asterisk ports

**Comment by gsanchietti** [comment]
Is this completed? Can we merge into #5074?
